learn2track/models/gru_regression.py

```diff
@@ -130,15 +129,7 @@ def use(self, X):
         directions = self.get_output(X)
         return directions
 
-    def get_init_states(self, batch_size):
-        states_h = []
-        for i, hidden_size in enumerate(self.hidden_sizes):
-            state_h = np.zeros((batch_size, hidden_size), dtype=floatX)
-            states_h.append(state_h)
-
-        return states_h
-
-    def make_sequence_generator(self, subject_id=0):
+    def make_sequence_generator(self, subject_id=0, **kwargs):
```

Comment:
I prefer using `**_` instead of `**args` to make it clear the additional arguments aren't going to be used.